workspaces/diff-engine/src/queries/endpoint.rs

```diff
@@ -16,9 +19,20 @@ impl<'a> EndpointQueries<'a> {
       endpoint_projection,
     }
   }
+
+  pub fn extract_normalized_path(interaction: &HttpInteraction) -> &str {
```

Comment:
If we end up going with this, lets not make this a public function until something outside `EndpointQueries` has a good reason to call this from outside.

Reply:
[] helper should be private

Reply:
- [ ] helper should be private